com.unity.perception/Runtime/GroundTruth/Labelers/Resources/KeyValuePanel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public class KeyValuePanel : MonoBehaviour
+    {
+        Text key = null;
+        Text value = null;
+
+        void Awake()
+        {
+            key = this.transform.Find("Key").GetComponent<Text>();
```

Comment:
Instead of Find, these references can be serialized in the panel. Or, you could create the objects at runtime. In its current state, this type has some very weakly expressed contracts around its child objects.

Reply:
Done.